custom_components/intellicenter/sensor.py

```diff
@@ -221,17 +221,17 @@ def device_class(self) -> Optional[str]:
         self._device_class
 
     @property
-    def state(self):
+    def state(self) -> str:
```

Comment:
The [docs](https://developers.home-assistant.io/docs/core/entity/sensor) say that state should be a string so I think we can just leave the value as a string instead of converting to int or float.